<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RaskGrade</class>
 <widget class="QDialog" name="RaskGrade">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../qrc/Top.qrc">
    <normaloff>:/new/wl/Logo_ico.png</normaloff>:/new/wl/Logo_ico.png</iconset>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>3</x>
     <y>2</y>
     <width>731</width>
     <height>571</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>330</y>
      <width>721</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>341</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbInheritRask">
        <property name="text">
         <string>继承前一版风险</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbHandAddRask">
        <property name="text">
         <string>手动增加风险</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pbDelHandRask">
        <property name="text">
         <string>删除手动增加风险</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>390</y>
      <width>731</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>6</x>
       <y>10</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>请指定FMEA风险评分专家：</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEditExpertList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>731</width>
       <height>151</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../qrc/Top.qrc"/>
 </resources>
 <connections/>
</ui>
